website/src/pages/docs/guides/react-vue.mdx

```diff
@@ -130,8 +130,9 @@ export default config
 **Usage with `@tanstack/react-query`**
 
 If you are using `@tanstack/react-query`, we recommend using it with `graphql-request@^5.0.0` to get the best developer experience.
-<br/>
-If you are willing to provide your own fetcher, you can directly jump to the ["Appendix I: React Query with a custom fetcher setup"](#appendix-i-react-query-with-a-custom-fetcher-setup) and continue the guide once React Query is properly setup.
```

Comment:
Not sure why this wasn't picked up in a previous CI/CD run. Should be fixed now :)

Reply:
Here's what this looks like in the website preview:

<img width="856" alt="Screenshot 2023-06-12 at 8 34 58 pm" src="https://github.com/dotansimha/graphql-code-generator/assets/33769523/66ded7df-4ee4-4097-bc1a-c3dcc1aed900">